src/session/DailyEventHandler.ts

```diff
@@ -59,11 +59,19 @@ export class DailyEventHandler {
   // onFatalError() handles Daily's "error" event
   onFatalError(dailyEvent: DailyEventObjectFatalError | undefined) {
     console.error("fatal error", dailyEvent);
-    const error = dailyEvent?.error;
+
+    const error: unknown = dailyEvent?.error;
```

Comment:
@lazeratops its because of the eslint/typescript update. The linter yells at you when you assign `any` to a new variable and `dailyEvent?.error` is of type `any`.

Typescript wants people to use `unknown` because it's safer, as it forces you to unbox the values, while `any` can leak through the codebase.